pkgs/test/haskell/ghcjs-eval/default.nix

```
@@ -0,0 +1,6 @@
+# This tests that we at least are able to evaluate a ghcjs-based derivation
+# to avoid accidental evaluation-related regressions. We can't test actually
+# building ghcjs, as it exceeds the output limit on Hydra.
+{ haskell, emptyFile }:
+
+builtins.deepSeq haskell.packages.ghcjs.hello.drvPath emptyFile
```

```suggestion
builtins.seq haskell.packages.ghcjs.hello.drvPath emptyFile
```

This will suffice and has no risk of divergence.